.github/workflows/sync-migrate.yml

```
@@ -18,7 +18,9 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Install pnpm
-        run: corepack enable pnpm
+        run: |
+        corepack enable
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 22: could not find expected ':'

(syntax-check)